FWCore/ServiceRegistry/interface/GlobalContext.h

```diff
@@ -76,6 +80,7 @@ namespace edm {
 
   void exceptionContext(std::ostream&, GlobalContext const&);
   std::ostream& operator<<(std::ostream&, GlobalContext const&);
+  void exceptionContext(cms::Exception&, GlobalContext const&, char const* context);
```

Comment:
On one hand taking `cms::Exception` instead of `std::ostream` is inconsistent and less general, but on the other hand I like the call side becoming shorter. @Dr15Jones, any thoughts?

Reply:
seems fine to me.